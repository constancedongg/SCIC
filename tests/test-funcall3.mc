int func main() {
    int a;
    a = 3;
    print(2);
    return 0;
}