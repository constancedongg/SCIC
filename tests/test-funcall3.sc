
float func foo(float x) {
    float a = x + 0.99;
    printf(x);
    printf(a);
    return a / 0.1 * 3.0;
}

int func main() {
    printf(foo(0.1));
    /* assign to a local*/
    float ua = foo(0.2);
    printf(ua); 
    return 0;
}