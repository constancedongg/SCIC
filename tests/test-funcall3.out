0.1
1.09
32.7
0.2
1.19
35.7